src/render.cr

```diff
@@ -10,18 +10,18 @@ module Artanis
     macro render(name, engine, layout = "layout")
       {% if layout %}
         render_{{ layout.id }}_{{ engine.id }} do
-          render_{{ name.id }}_{{ engine.id }}
+          render_{{ name.gsub(/\//, "__SLASH__").id }}_{{ engine.id }}
         end
       {% else %}
-        render_{{ name.id }}_{{ engine.id }}
+        render_{{ name.gsub(/\//, "__SLASH__").id }}_{{ engine.id }}
       {% end %}
     end
 
     macro views_path(path)
       {%
-        views = `cd #{ path } 2>/dev/null && ls *.ecr || echo -n ""`
+        views = `cd #{ path } 2>/dev/null && find . -name "*.ecr" -type f | cut -c 3- || true`
```

Comment:
`find` prints a relative path with a leading `./`; `cut -c 3-` removes the `./` by selecting the text after (and including) the third character to the end of the line. You could do the same in Crystal with `.gsub(/^\.\//, "")`.